WebApp/autoreduce_webapp/reduction_viewer/views.py

```diff
@@ -339,6 +342,8 @@ def instrument_summary(request, instrument=None):
         if len(runs) == 0:
             return {'message': "No runs found for instrument."}
 
+        has_variables = InstrumentVariablesUtils().get_default_variables(instrument_obj.name)
```

Comment:
Is there a bool() missing here around InstrumentVariablesUtils().get_default_variables(instrument_obj.name)?